solutions/agents/provider.tf

```diff
@@ -20,8 +20,13 @@ provider "helm" {
   }
 }
 
+data "ibm_container_vpc_cluster" "cluster" {
+  name      = var.cluster_name
+  wait_till = "normal"
```

Comment:
let's expose `wait_till` as a tf input variable and please add variable validation as well, [the valid values are](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/container_vpc_cluster) :
    ```
    Normal: 
    MasterNodeReady: 
    OneWorkerNodeReady: 
    IngressReady: 
    ```

similar code https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/26852302327924b2fcf190a9976ee8d5603715fc/variables.tf#L942